docker-compose-swarm.yml

```diff
@@ -5,7 +5,7 @@ services:
     ports:
       - "2181:2181"
   kafka:
-    image: ${REPO}kafka-docker:latest
+    image: ${REPO}kafka:latest
```

Comment:
i'm not sure why this even uses the `${REPO}` environment var. It should just be the canonical `wurstmeister/kafka:latest` image

Reply:
I think it's there as a leftover of some tests using docker stacks. In stacks you usually build a temporal registry, and you have to tag the images something like hostname:port/kafka (e.g. 127.0.0.1:5000/kafka). Anyway, once the image is stable and available in the docker hub, no need for this REPO variable, so it should be removed in the github distribution.